app/src/main/res/values/strings.xml

```diff
@@ -122,6 +122,10 @@ Home Assistant instance</string>
   <string name="label_icon">Icon:</string>
   <string name="label_label">Label:</string>
   <string name="label_service">Service:</string>
+  <string name="lang_option_label_default">Device Language</string>
+  <string name="lang_option_value_default">default</string>
+  <string name="lang_summary_settings">Changing default language may have influence on device entities naming</string>
```

Comment:
Should we also mention that this setting is independent of the User Profile language option?

Reply:
I'm not sure. After this changes we will have the same lang for all activities, including frontend language. So basicaly this setting override profile language as well.

Reply:
When I say frontend I refer to the HA frontend that is visible in our Webview activity.  The webview is unaware of the devices language and is also unaware of the apps language.  We don't send this data to the HA frontend so it will never be aware of this change.  Load up the APK and test it for yourself.  You will see the HA frontend when you back out of settings is not updated.

Reply:
Yep, I meant it as well. I'm not sure how, but it follows the selected app language. I just don't know a good way to recreate webview activity when we back out of settings. Just reopen the application and you will see the new langg applied to frontend.

Reply:
Ok I see that now, having to restart the app isn't immediately apparent since that is what needs to be happen.  I wonder if we should add an alert telling the user we need to restart once they change the language and perform the restart if they accept.  Users may get surprised if they back out and find HA in one language and then suddenly it changes when the app got closed for some reason 😃

Reply:
We do not need to restart the whole app but only webview activity. Like the settings activity is recreates.

@JBassett what is the prefered way to notify webview activity about setting changed? I'm not sure about StartActivityForResult callback or listening for shared preference cahange?

Reply:
I think changing the theme in settings does something similar right?

Reply:
Not exactly. AppCompatDelegate is doing all work for themes applying.

Reply:
In the `onResume` maybe store the current locale then check on every `onResume` to see if it changed, if it did call `recreate()`?

Reply:
Yep, works ok.